File: content-interface/src/main/java/com/service/OrderLogService.java
package com.service;

import com.shopxx.pojo.XxOrderLog;
import entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface OrderLogService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<XxOrderLog> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(XxOrderLog orderLog);
	
	
	/**
	 * 修改
	 */
	public void update(XxOrderLog orderLog);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public XxOrderLog findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(XxOrderLog orderLog, int pageNum, int pageSize);
	
}

File: manager-web/src/main/webapp/js/service/orderLogService.js
//服务层
app.service('orderLogService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../orderLog/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../orderLog/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../orderLog/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../orderLog/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../orderLog/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../orderLog/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../orderLog/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
